### PR TITLE
Changelog for 8.15.1

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by SkySkimmer** [comment]
AFAICT we don't do a release summary for patch versions.

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -94,6 +94,106 @@ This release is the result of 384 merged PRs, closing 67 issues.
 | Nantes, January 2022,
 | Matthieu Sozeau for the Coq development team
 
+Changes in 8.15.1
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:97]: We usually insert this after the "Changes in 8.15.0" (I don't mind using a truly reverse chronological order, but in this case, we should update the whole changelog to be consistent).

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -94,6 +94,106 @@ This release is the result of 384 merged PRs, closing 67 issues.
...
+  inconsistency when using module subtyping with squashed inductives
+  (`#15839 <https://github.com/coq/coq/pull/15839>`_,
+  fixes `#15838 <https://github.com/coq/coq/issues/15838>`_ (reported by Pierre-Marie Pédrot),
+  by Gaëtan Gilbert).
+
+Notations
+^^^^^^^^^
+
+- **Fixed:**
+  Check for prior declaration of a custom entry was missing for motations in only printing mode
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:122]:
```suggestion
  Check for prior declaration of a custom entry was missing for notations in only printing mode
```

File: doc/sphinx/changes.rst [hunk 3]
```diff
@@ -94,6 +94,106 @@ This release is the result of 384 merged PRs, closing 67 issues.
...
+  by Jim Fehrle).
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+- **Fixed:**
+  Ensure that the names of arguments of inductive schemes are distinct
+  so that the new Coq 8.15 preservation of argument names in the ``with``
+  clause of tactics in `#13837 <https://github.com/coq/coq/pull/13837>`_
+  work as in Coq 8.14 for these schemes
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:191]:
```suggestion
  works as in Coq 8.14 for these schemes
```

**Comment by ejgallego** [comment]
It also seems to me that we didn't require a changelog for each bug fixed. Seems like a good thing to do, WDYT?

File: doc/sphinx/changes.rst (hunk 1 above)
Comment (SkySkimmer) [reply doc/sphinx/changes.rst:97]: I hate the mixed order but fixing it is a lot of work, let's just drop consistency.

Comment (Zimmi48) [reply doc/sphinx/changes.rst:97]: Sorry then for introducing the mixed order (I think it's my fault). But I'd like not to be alone approving this change (to make sure that we're consistent from then on).

Comment (SkySkimmer) [reply doc/sphinx/changes.rst:97]: >Sorry then for introducing the mixed order (I think it's my fault).

I doubt it, it's been like that for the whole changelog (eg V8 goes `8.0beta` then `8.0`)

Comment (SkySkimmer) [reply doc/sphinx/changes.rst]: On second thought let's be consistent, I'll take the time to fix it all for 8.16

**Comment by SkySkimmer** [comment]
>It also seems to me that we didn't require a changelog for each bug fixed. Seems like a good thing to do, WDYT?

What do you mean?

File: doc/sphinx/changes.rst (hunk 1 above)
Comment (Zimmi48) [reply doc/sphinx/changes.rst]: Thanks!

**Review by Zimmi48 — APPROVED** [review]
@coqbot merge now

**Comment by Zimmi48** [comment]
> It also seems to me that we didn't require a changelog for each bug fixed. Seems like a good thing to do, WDYT?

In my opinion, there is always a blurry limit between which user-visible changes are worth documenting and which ones are not (e.g., you would not document a typo fix, but also probably not a minor error message improvement). One thing that should clearly not get into the (user) changelog is anything that is not user-visible, in particular any fix to a bug that was never released (such as in a new feature).

**Comment by Zimmi48** [comment]
I'm fine with this, although it still seems blurry to me (where you put the limit of what a "bug" is). Anyway, feel free to open a PR changing the contributing guide and PR template to propose this (for future releases).

**Comment by ejgallego** [comment]
> I'm fine with this, although it still seems blurry to me (where you put the limit of what a "bug" is). Anyway, feel free to open a PR changing the contributing guide and PR template to propose this (for future releases).

Indeed, it seems worth to me so I'll add this to my todo list.

I'd propose to consider something a bug what is tagged with tag `kind: bug`, what the tag is, that's indeed open to interpretation but should be a bit orthogonal of the changelog stuff.

**Pull request merged**